src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { tabs, selectedTabId, onTabSelected } = props;
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
```suggestion
export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

OR

```suggestion
export const Tabs: React.FC<Props> = ({ 
  tabs, 
  selectedTabId,
  onTabSelected 
}) => {
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Reply:
Thanks for the suggested changes to optimize the code.